pypeman/channels.py

```diff
@@ -377,95 +383,218 @@ def handle_and_wait(self, msg):
         """
         return self.loop.run_until_complete(self.handle(msg))
 
+    def _has_callback(self):
+        return hasattr(self, "_callback")
+
+    async def _call_init_nodes(self, msg, start_nodename=None):
+        if self.init_nodes:
+            if start_nodename:
+                node = self.get_node(start_nodename)
+                if node not in self.init_nodes:
+                    raise ValueError("Node named %s not in init nodes", start_nodename)
```

Comment:
does this really work? I don't think valueError interpolates strings.

so probably better:
`raise ValueError(f"Node named {start_nodema,e} not in init nodes")`

or

`raise ValueError("Node named %s not in init nodes" % start_nodename)`

Reply:
done